dockerfile_parse/parser.py

```diff
@@ -494,6 +494,8 @@ def _modify_instruction(self, instruction, new_value):
                 del lines[insn['startline']:insn['endline'] + 1]
                 lines.insert(insn['startline'], new_line)
                 self.lines = lines
+                if insn['instruction'] == 'FROM':  # Only overwrite first base-image
```

Comment:
Could be `if instruction == 'FROM':`, and that might make the intent clearer too.

Reply:
Fixed.